pipeline/trusted-resources/cmd/sign/sign_test.go

```diff
@@ -77,44 +86,81 @@ var (
 	}
 )
 
-func TestSignTask(t *testing.T) {
+func TestSign(t *testing.T) {
```

Comment:
Do we need to add tests that fail to verify? i.e. modify it after signing?

Reply:
That is a good point! 
But that case is covered in another folder, so here we just need to make sure the signed files can pass verification. That's my opinion

Reply:
SG!